### DISCUSSION TITLE
problem running milvus 1.1.1 gpu in wsl2

### DISCUSSION THREAD

**Discussion by YangIsNotAvailable** [milvus-io/milvus#6875] [Q&A and General discussion]
I have all the cuda drivers/toolkit set up as instructed in https://docs.nvidia.com/cuda/wsl-user-guide/index.html and I can successfully run the example N-body container. But I still have trouble run the gpu milvus, and this is the error msg I got

docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: Running hook #0:: error running hook: exit status 1, stdout: , stderr: nvidia-container-cli: mount error: file creation failed: /var/lib/docker/overlay2/1a7b9988a0557f2270a60f21018f66840ec14d2f1a347768e3c86a004204da0d/merged/usr/lib64/libnvidia-ml.so.1: file exists: unknown.

Thank you in advance for help.

**Comment by yhmo** [comment]
I don't have experience on WSL. Just search "nvidia-container-cli: mount error: file creation failed" on Google, looks like relate to usage of nvidia-docker:  https://github.com/NVIDIA/nvidia-docker/issues/825
